### ISSUE TITLE
available count is: 0 on deploys

### ISSUE THREAD

**Issue by camallen** [zooniverse/designator#33]
seems the (re)loader doesn't kick in on deploy and people complain as it fails over, this may be a feature though. Just noting that it happened a few times round deploy.

**Comment by marten** [comment]
Yes, and should be caught by Panoptes doing its own selection when Designator comes back with nothing. Same behaviour also applies during normal operation when a workflow hasn't been used for a while and is unloaded, just less visibly than on deploy.
